src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -77,7 +77,7 @@ Author(s):
     X(CloseOnExitMode, CloseOnExit, "closeOnExit", CloseOnExitMode::Automatic)                                                                                 \
     X(hstring, TabTitle, "tabTitle")                                                                                                                           \
     X(Model::BellStyle, BellStyle, "bellStyle", BellStyle::Audible)                                                                                            \
-    X(bool, UseAtlasEngine, "experimental.useAtlasEngine", false)                                                                                              \
+    X(bool, UseAtlasEngine, "experimental.useAtlasEngine", Feature_AtlasEngineByDefault::IsEnabled())                                                          \

```

Comment:
I was thinking we could use the existing Feature_AtlasEngine and remove the other feature gating at the same time..!

Reply:
_DERP_ I didn't even notice we had that feature flag already hahaha